README.md

```diff
@@ -32,7 +80,7 @@ continuous integration, code coverage, nightly release, conformance and etc.
 
 Our E2E testing uses
 [kubetest](https://github.com/kubernetes/test-infra/blob/master/kubetest) to
```

Comment:
We are now using kubetest2 instead - https://github.com/kubernetes-sigs/kubetest2, could you update it here? Thanks!